src/components/StripeForm.js

```diff
@@ -0,0 +1,261 @@
+import React, { useState, useEffect } from "react"
+import { useStripe, useElements, CardElement } from "@stripe/react-stripe-js"
+// import { Formik, Field, Form, ErrorMessage } from "formik"
+// import * as Yup from "yup"
+
+export default function StripeForm() {
+  const stripe = useStripe()
+  const elements = useElements()
+
+  const [succeeded, setSucceeded] = useState(false)
+  const [error, setError] = useState(null)
+  const [processing, setProcessing] = useState("")
+  const [disabled, setDisabled] = useState(true)
+  const [clientSecret, setClientSecret] = useState("")
+  const [donationAmount, setDonationAmount] = useState({
+    activeIndex: -1,
+    value: "",
+    checked: false,
+  })
+  const [firstName, setFirstName] = useState("")
+  const donationAmounts = [
+    {
+      label: "50",
+      value: "5000",
+      text: "Support those in need!",
+    },
+    {
+      label: "100",
+      value: "10000",
+      text: "Move the needle for an artist",
+    },
+    {
+      label: "500",
+      value: "50000",
+      text: "Support 1 month for 1 artist",
+    },
+    {
+      label: "1500",
+      value: "150000",
+      text: "Support an artist for the full 3 months of support",
+    },
+    {
+      label: "custom",
+      value: "custom",
+      text: "Enter custom amount",
+    },
+  ]
+
+  console.log(donationAmount)
+  console.log(firstName)
```

Comment:
I would avoid checking in any console.log action... they are viable for local testing, but probably should not be included in commits